addons-cluster/etcd/values.schema.json

```diff
@@ -0,0 +1,216 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "tolerations": {
+      "title": "Tolerations",
+      "description": "Define global Tolerations for the cluster all pod's assignment",
+      "type": "array",
+      "items": {
+        "type": "object"
+      },
+      "default": []
+    },
+    "topologySpreadConstraints": {
+      "title": "Topology Spread Constraints",
+      "description": "Topology Spread Constraints for pod assignment spread across your cluster among failure-domains.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "default": ["kubernetes.io/hostname"]
+    },
+    "affinity": {
+      "title": "Affinity",
+      "description": "Affinity setting for etcd cluster pods assignment",
+      "type": "object",
+      "default": {}
+    },
```

Comment:
too much, if users want to customize these properties, they may write a Cluster yaml.